### PR TITLE
Validator ROI & balance

### PR DESCRIPTION
**Pull request by morrigan**
Status (uptime) will be resolved with #213 

Also did some refactoring as ValidatorSimple is not simple anymore..

### CODE REVIEW

**Comment by morrigan** [comment]
@mpetrunic Actually, if we don't need anything except balance from validator, I'll remove this additional redux reducer. I just realized that it might be redundant

File: src/renderer/actions/validator.ts [hunk 1]
```diff
@@ -1,24 +1,59 @@
...
+}
+
+export const loadValidatorsAction = () => {
+    return async (dispatch: Dispatch<Action<unknown>>, getState: () => IRootState): Promise<void> => {
+        const auth = getState().auth;
+        if (auth && auth.account) {
+            const validators = auth.account.getValidators();
+            const validatorArray = validators.map((v) => ({
+                name: auth.account!.name,
+                status: "TODO status",
```
Comment (mpetrunic) [inline src/renderer/actions/validator.ts:21]: This already exists, you should probably look into integrating in some future PR

File: src/renderer/actions/validator.ts [hunk 2]
```diff
@@ -1,24 +1,59 @@
...
+                publicKey: v.publicKey.toHexString(),
+                deposit: 30,
```
Comment (mpetrunic) [inline src/renderer/actions/validator.ts:23]: This can also be fetched from node instead of stubbed

File: src/renderer/actions/validator.ts [hunk 3]
```diff
@@ -1,24 +1,59 @@
...
+
+            dispatch({
+                type: ValidatorActionTypes.LOAD_VALIDATORS,
+                payload: validatorArray,
+            });
+        }
+    };
+};
+
+export const loadValidatorBalance = (validator: string) => {
```
Comment (mpetrunic) [inline src/renderer/actions/validator.ts:36]: Might be better if we fetch balance of multiple validators, which allows us to use single api request to fetch them all later on

File: src/renderer/actions/validator.ts (hunk 2 above)
Comment (morrigan) [reply src/renderer/actions/validator.ts:23]: I am not sure that we even need this info? Maybe we will when we support multiple deposits with values lower than 32 eth but currently this is not used and can remove it

File: src/renderer/actions/validator.ts (hunk 1 above)
Comment (morrigan) [reply src/renderer/actions/validator.ts:21]: Created #215

File: src/renderer/actions/validator.ts (hunk 2 above)
Comment (mpetrunic) [reply src/renderer/actions/validator.ts:23]: You can deposit more than 32 ethers though, but yeah, we can probably remove for now

File: src/renderer/components/Cards/ValidatorStat.tsx [hunk 4]
```diff
@@ -33,20 +34,18 @@ const renderUptime = (props: IValidatorStatProps): React.ReactElement => {
...
 };
 export const ValidatorStat: React.FunctionComponent<IValidatorStatProps> = (
     props: IValidatorStatProps) => {
-    
+
     switch(props.type){
         case "ROI": return renderROI(props);
         case "ETH": return renderBalance(props);
-        case "Uptime": return renderUptime(props);
+        case "Status": return renderUptime(props);
```
Comment (mpetrunic) [inline src/renderer/components/Cards/ValidatorStat.tsx:48]: ?

Comment (morrigan) [reply src/renderer/components/Cards/ValidatorStat.tsx:48]: Uptime in design was by days so I guess this makes more sense?

**Pull request merged**